doc/users/next_whats_new/url_in_images_for_pdfs.rst

```diff
@@ -0,0 +1,10 @@
+URL Support for Images in PDF Backend
+-------------------------------------
+
+The PDF backend can now generate clickable images if a URL is provided to the
+image. There are a few limitations worth noting though:
+
+* If parts of the image are clipped, the non-visible parts are still clickable.
```

Comment:
Does this mean, if you zoom in on the image, parts outside the Axes will be clickable?

Reply:
Yes. It is sort of easy to fix for non-transformed images (if there is a way to get the Axes extents from the gc(?). But for transformed it is much harder.

(I think the same holds for texts.)